source/index.html.haml

```diff
@@ -83,17 +84,34 @@ title: RubyConf Colombia 2017
 
       .newsletter
         .newsletter__description
-          If you want to receive info about the conference, join our mailing list.
-          %br
-          No spam, we promise.
+          If you want to receive info about the conference, join our mailing list.No spam, we promise.
         %form.form.row.newsletter__form.mailchimp-form{:action => "//rubyconf.us11.list-manage.com/subscribe/post?u=7b7c2e5ead7427750ced71f09&id=e7b035683b", :enctype => "multipart/form-data", :method => "post", :name => "mc-embedded-subscribe-form", :target => "_blank"}
           .newsletter__field
             %input#EMAIL.newsletter__field_input{:name => "EMAIL", :placeholder => "your@email.com", :type => "email"}/
             %button.newsletter__field_button Subscribe
-      .pastEvents
-        %p Previous events
-        %a{:href => "http://2016.rubyconf.co", :target => "_blank"} 2016
-        %a{:href => "http://2015.rubyconf.co", :target => "_blank"} 2015
+      .speakers
+        %h4.speakers__title Speakers:
+        %ul.speakers__profiles
+          %li.speakers__item
+            %a.speakers__link{:href => "https://twitter.com/eileencodes", :target => "_blank"}
+              %img{:src => "./assets/images/speakers/Eileen-m-Uchitelle.jpeg", :alt => "Eileen M. Uchitelle"}
+              %p.profile_woodroffe
+                Eileen
+              %p.profile_woodroffe
+                M. Uchitelle
+          %li.speakers__item
+            %a.speakers__link{:href => "https://twitter.com/konstantinhaase", :target => "_blank"}
+              %img{:src => "./assets/images/speakers/Konstantin.jpg", :alt => "Konstantin"}
```

Comment:
we prolly want to use the full name `Konstantin Haase` (as we did with the other speakers)